### PR TITLE
Lps 154104

### PR DESCRIPTION
**Pull request by SamZiemer**
Forwarding from: https://github.com/liferay-ts-reviewer/liferay-portal/pull/66

### CODE REVIEW

**Comment by gabrielwas** [comment]
@danvb112 can you check it, please?

File: modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectView/ModalAddDefaultSortColumn/ModalAddDefaultSortColumn.tsx [hunk 1]
```diff
@@ -96,7 +96,11 @@ export function ModalAddDefaultSortColumn({
 	const onSubmit = (event: FormEvent) => {
 		event.preventDefault();
 
-		const objectFieldName = selectedObjectSortColumn?.objectFieldName;
+		let objectFieldName = selectedObjectSortColumn?.objectFieldName;
+
+		typeof objectFieldName === 'undefined' &&
+			filtredObjectSortColumn.length > 0 &&
+			(objectFieldName = filtredObjectSortColumn[0].objectFieldName);
```
Comment (danvb112) [inline modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectView/ModalAddDefaultSortColumn/ModalAddDefaultSortColumn.tsx:103]: Hey @thienquach87 and @SamZiemer, I think we can make this code more readable, keeping your logic, you can add this in an `if` condition, like:

```ts
if (!objectFieldName && !!filtredObjectSortColumn.length) {
	objectFieldName = filtredObjectSortColumn[0].objectFieldName;
}
```

**Review by danvb112 — CHANGES_REQUESTED** [review]
Hey @thienquach87 and @SamZiemer, good job on this solution, I have only one comment about it :smiley:

File: modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectView/ModalAddDefaultSortColumn/ModalAddDefaultSortColumn.tsx (hunk 1 above)
Comment (thienquach87) [reply modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectView/ModalAddDefaultSortColumn/ModalAddDefaultSortColumn.tsx:103]: Hi, @SamZiemer could you help adjust the code as @danvb112 suggested on this PR directly? I verified this logic is similar to my idea.

Comment (thienquach87) [reply modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectView/ModalAddDefaultSortColumn/ModalAddDefaultSortColumn.tsx:103]: @danvb112 I created new PR, please help check it
https://github.com/liferay-objects/liferay-portal/pull/807

**Comment by danvb112** [comment]
Sent to: https://github.com/liferay-objects/liferay-portal/pull/807

**Pull request closed without merging**